likecoin/admin/likecoin.php

```diff
@@ -83,5 +109,6 @@ function likecoin_add_admin_hooks() {
 	add_action( 'admin_notices', 'likecoin_show_admin_errors' );
 	add_action( 'manage_posts_columns', 'likecoin_add_posts_columns', 10, 2 );
 	add_action( 'manage_posts_custom_column', 'likecoin_populate_posts_columns', 10, 2 );
+	add_action( 'admin_enqueue_scripts', 'likecoin_show_plugin' );
```

Comment:
I think it's even more confusing if we use add_meta_boxes, and then actually adds the sidebar not meta_boxes?
Since likecoin_show_plugin = metabox or sidebar.

Reply:
will use add_meta_boxes after discussion offline